js/RateCurve.js

```diff
@@ -1,8 +1,12 @@
 'use strict';
```

Comment:
You should probably add a note to the top of this file stating that it can only be used if PID_ADVANCED_CONFIG, RX_CONFIG, MISC have been loaded, to avoid people trying to use it in places where this isn't the case. (This is a difficult to test thing anyway, since it obviously depends on what tabs have been loaded before the current one.)